requirements-test.txt

```diff
@@ -4,7 +4,7 @@ tox
 
 flake8<3.0
 
-pytest==3.2.5
+pytest>=4.3,<4.4
```

Comment:
Does this need an upper limit? pytest is now on 5.2.2.

Reply:
It certainly doesn't.